packages/core/package.json

```diff
@@ -52,6 +52,7 @@
     "@faker-js/faker": "^8.4.1",
     "@latitude-data/compiler": "workspace:^",
     "@latitude-data/env": "workspace:*",
+    "@latitude-data/constants": "workspace:*",
```

Comment:
Next step in another moment is to move all constants that are from
1. Vercel AI
2. Compiler 

Into constants so sharing this types between `gateway` and `sdk` and `core` is perfect and no one depends on the others